src/less/blocks/widget.less

```diff
@@ -0,0 +1,209 @@
+.widget {
+  min-width: 280px;
+  max-width: 390px;
+  width: 100%;
+  margin: 27px auto;
+  display: flex;
+  flex-direction: column;
+  
+  &:not(:first-child) {
+    display: none;
+  }
+}
+
+.widget__header {
+  background-color: #fff;
+  border-radius: 10px 10px 0 0;
+  padding: 17px;
+  height: 54px;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.widget__refresh-btn {
+  position: relative;
+  border: none;
+  cursor: pointer;
+  
+  &::before {
+    content: "";
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 24px;
+    height: 21px;
+    background-image:
+      url("../img/refresh.svg");
+  }
+}
+
+.widget__body {
+  background-color: #fff;
+  padding: 187px 0 17px;
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  
+  &::before {
+    content: "";
+    display: inline-block;
+    position: absolute;
+    top: 14px;
+    left: 3px;
+    right: 0;
+    margin: 0 auto;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 148px;
+    height: 123px;
+    background-image:
+      url("../img/cloudy-skies.svg");
```

Comment:
Just a thought - maybe it would be better to make the main icon as a separate component (not another component's background). I think it'll be way easier to maintain. No need to fix it in this homework though.

Reply:
Considered this option but thought that it's just a decoration, not a content image